pom.xml

```diff
@@ -78,6 +78,7 @@
 
     <mavenVersion>3.5.0</mavenVersion>
     <atlasVersion>0.17.1</atlasVersion>
+    <daVersion>2.1.0-SNAPSHOT</daVersion>
```

Comment:
Is there not a non-snapshot version to use?

Reply:
Not yet. I was thinking about separating the API from DA and release it once finalized for PNC 2.1 even though we will be still in development phase. Have to discuss that with the team.

Reply:
OK, we will need to discuss this because right now it didn't even get to the tests as it isn't picking up the snapshot.